_posts/2016-03-14-public-speaking-part2-where-to-start.markdown

```diff
@@ -0,0 +1,69 @@
+
+If you have a meet-up with a similar positive vibe, there’s no reason not to have a chat with the organisers and see what the next steps would be. It might even be well worth practicing a talk here, for when you’ve got a bigger event coming up and you would like feedback. Bare in mind, the audience wants you to succeed!
+
+Sidenote — if your city does not (yet) have a great meet-up, it’s well worth starting one.
+
+
+### Conferences
+
+Often considered the most intimidating of all. People have purchased tickets and possibly travelled from far. The audience is usually in the hundreds, occasionally even more than a thousand. Even the most experienced speakers can get a bit nervous for this.  
+
+Funnily enough, my first ever talk, since starting out as a developer, was at a [conference][iosdevuk]. It happened to have a very friendly atmosphere and really wasn’t that scary. I can’t stress enough that, yes, the audience wants you to succeed. I remember talking through my introduction and seeing all the curious, attentive faces. They were there because they were interested. They cared for what I had to say.
+
+I will say that this doesn’t necessarily mean you should go talk about anything your heart desires. Ultimately the attendees are there to learn, so it’s only fair that as speakers you’d tailor your presentation towards your audience. But more on that in Part 3.
+
+To be considered, often conference organisers ask people to submit a CfP (Call for Papers / Call for Proposals). This is your chance to submit details on what you’d like to present on. You usually want to include a preliminary talk title, an abstract, and a bit of background on yourself. If you’ve done any talks before and they happen to have been recorded, it’s worth including the video or otherwise previous slidedecks.
+
+As for how CfPs then get chosen depends on the conference. Most of the time the organisational committee selects talks from the proposal list. Some conferences have [an independent panel][altconf] review anonymised proposals or have a system in place where [the community votes][uikonf]. Once it’s been decided, they will get back in touch with you.
+
+Usually conferences are swarmed with CfPs so please don’t be discouraged if it takes a few tries to get picked. It’s understandable but bare in mind that a friend of mine felt like this 2 years ago also. Yet he kept at it and got invited to NSSpain 2014. He got incredibly active in the SF Swift meet-up scene and open source. All of this got the ball rolling and now he's given talks all around the world. [You might even have heard of him][twt_jp].
```

Comment:
This sentence feels like it should be broken up a bit `It’s understandable but bare in mind that a friend of mine felt like this 2 years ago also`

Perhaps?

> It’s understandable to feel rejected and give up, but bare in mind that a friend of mine felt this way yet kept applying. He got his break at NSSpain 2014, and now is incredibly active in the [San Francisco Swift](http://www.meetup.com/swift-language/) meet-up scene.  All of this got the ball rolling and now he's given talks all around the world.